python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -119,32 +119,18 @@ def run(self, inputCat, wcs, calib):
         return outputCat
 
 
-class RunTransformConfig(pexConfig.Config):
-    """!Configuration for RunTransformTask."""
+class RunTransformConfigBase(object):
+    """!Configuration for RunTransformTaskBase derivatives."""
     transform = pexConfig.ConfigurableField(
         doc="Subtask which performs transformations",
         target=TransformTask
     )
-    measConfig = pexConfig.Field(
-        dtype=str,
-        doc="Dataset type of measurement operation configuration",
-        default="processCcd_config"
-    )
-    measType = pexConfig.ChoiceField(
-        dtype=str,
-        doc="Type of measurement operation performed",
-        default="SingleFrame",
-        allowed={
-            "SingleFrame": "Single frame measurement",
-            "Forced": "Forced measurement"
-        }
-    )
 
 
-class RunTransformTask(pipeBase.CmdLineTask):
+class RunTransformTaskBase(object):
```

Comment:
My worry was that it's abstract-ness is not immediately clear: since we're providing a ``run()`` method, it looks like something you could pick up and use straight away as a command line task, but it wouldn't work.

I'm not particularly attached to this way of doing things though; I'm happy to just use single inheritance if you prefer.